### PR TITLE
wrap text properly in log display

### PR DESCRIPTION
**Pull request by ramyaparimi**
This PR is for [issue#62](https://github.com/fedora-infra/mote/issues/62) to fix the word wrapping issue.

### CODE REVIEW

**Comment by nasirhm** [comment]
@ramyaparimi please add a screen shot of the change. (as It's a frontend one).

How about using `word-break: break-all;` instead of `word-break: keepall-all;`
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

**Comment by ramyaparimi** [comment]
@nasirhm Thank you! I am happy about my first PR 😃 
And also thank you for suggesting screen shots. 
I personally feel `keep-all`  is wrapping words correctly and giving a better reading experience. I think `break-all` would be ideal when dealing with really long words, but I feel such very long words would probably not be used in meeting chats much. But, I might be wrong.. so please let me know what you all think!

**This is with the current format. `white-space: pre-wrap;` alone. The words are not wrapped properly which is making it difficult to read.**
image



**`white-space: pre-wrap; word-break:break-all;` is still breaking words inconveniently.**
image




**`white-space: pre-wrap; word-break:keep-all;` is wrapping words correctly giving a better experience to readers!**
image

File: mote/static/css/mote.css [hunk 1]
```diff
@@ -54,7 +54,7 @@
   width: 80%;
 }
 
-.log-display pre { white-space: pre-wrap; }
+.log-display pre { white-space: pre-wrap; word-break: keep-all; }
```
Comment (t0xic0der) [inline mote/static/css/mote.css:57]: The last preview at https://github.com/fedora-infra/mote/pull/147#issuecomment-721194503 - had both `pre-wrap` and `keep-all` as `white-space` and `word-break` configs. Looks good to me. :smile:

**Review by t0xic0der — APPROVED** [review]
A :+1: from my end.

**Comment by nasirhm** [comment]
> I personally feel `keep-all` is wrapping words correctly and giving a better reading experience. I think `break-all` would be ideal when dealing with really long words, but I feel such very long words would probably not be used in meeting chats much. But, I might be wrong.. so please let me know what you all think!

Hmm, makes sense.

`break-all` does seem the way to go :) 

Thank you for working on it.

**Review by nasirhm — APPROVED** [review]
LGTM.

**Review by nasirhm — APPROVED** [review]
LGTM, merging (Once, CI runs successfully).

**Comment by ramyaparimi** [comment]
@nasirhm Could you please check if this can be merged now?

**Comment by nasirhm** [comment]
> @nasirhm Could you please check if this can be merged now?

Sure, the CI ran successfully, can you please squash all the commits into one ?

**Comment by ramyaparimi** [comment]
Would these changes take time to reflect? I was checking one of the logs and looks like the issue still exists!  The new changes are not present in the inspect element either.

**Comment by nasirhm** [comment]
> Would these changes take time to reflect? I was checking one of the logs and looks like the issue still exists! The new changes are not present in the inspect element either.

Ramya, Actually the modifications and development we're doing here is not deployed yet, once we modernize the app a bit, We can take an issue in a sprint to work on the deployment for our development changes to reflect on the website.

**Pull request merged**